### DISCUSSION TITLE
Owner, group and autorisations for /ws-certs.d/

### DISCUSSION THREAD

**Discussion by Amachako** [cockpit-project/cockpit#16141] [Q&A] [answered]
Hello,

I kinda messed up with my cockpit install. I changed the ownership (and group) with the differents autorisations of the folder:
/etc/cockpit/ws-certs.d/ 

I want to know what are the default value of it so I can turn it back (I didn't note them sadly before). I'm forced to use 777 until I know what I must use and that's not safe.

If you can provide me the user:group and permissions this folder and the default file have that will save me.

**Comment by marusak** [comment, marked as answer]
```
$ ls -l /etc/cockpit/ws-certs.d/
total 8
-rw-r--r--. 1 root root       2171 Jul  1  2020 0-self-signed-ca.pem
-rw-r-----. 1 root cockpit-ws 3448 Jul  1  2020 0-self-signed.cert
```

**Comment by marusak** [comment]
```
 ls -l /etc/cockpit
total 8
-rw-r--r--. 1 root root    0 Nov 10  2020 cockpit.conf
drwxr-xr-x. 2 root root 4096 Jun 23 12:37 machines.d
drwxr-xr-x. 2 root root 4096 Jun 23 12:37 ws-certs.d
```

**Comment by martinpitt** [comment]
@Amachako : If you did not do any actual config change or have a custom certificate, you can also just remove all the files there. The next startup of cockpit will create a fresh self-signed certificate with correct permissions.
